templates/bootstrap_rpm/rhsm-katello-reconfigure.erb

```diff
@@ -0,0 +1,115 @@
+KATELLO_SERVER=<%= @rhsm_hostname %>
+KATELLO_SERVER_CA_CERT=<%= scope['certs::server_ca_name'] %>.pem
+KATELLO_DEFAULT_CA_CERT=<%= scope['certs::default_ca_name'] %>.pem
+KATELLO_CERT_DIR=<%= scope['certs::katello::rhsm_ca_dir'] %>
```

Comment:
I'm not a fan of this linking because it can easily break if we start to refactor things. Any thoughts on switching to EPP and explicitly passing it as context? It's more verbose, but we'll notice easily if it does break.